lib/haml_coffee_assets/action_view/resolver.rb

```diff
@@ -8,7 +8,7 @@ module ActionView
     # in it would normally be a fallback for all formats.
     #
     class Resolver < ::ActionView::FileSystemResolver
-      def find_templates(name, prefix, partial, details)
+      def find_templates(name, prefix, partial, details, outside_app_allowed = false)
```

Comment:
This would break versions of Rails that still use 4 arguments, right? I think this would either need to be behind a version guard, or changed to use `find_templates(*args)` as long as haml_coffee_assets doesn't really care internally what this method signature looks like.